### PR TITLE
Change tinymce lang when changeLanguage event is fired

### LINKED ISSUE #16646: In supplier the language does not change on the whole page
**Describe the bug**
When I change language in area description, the change isn't apply to the whole page

**To Reproduce**
1. Go to back office 
2. Click on catalog -> brands & suppliers -> suppliers add new supplier
3. Switch language in the description area 

**Screenshots**
https://drive.google.com/file/d/1GIoKlFsVbOsoNjfeEUr6fniH24l1z-ts/view

**Additional information**
PrestaShop version: 1.7.7.

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | When I change language in area description, the change isn't apply to the whole page, tinymce mainly
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #16646 
| How to test?  | Steps to reproduce the behavior:<br>- Go to back office<br>- Click on catalog -> brands & suppliers -> suppliers add new supplier<br>- Switch language in the description area<br>- See if tinymce lang changed too

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17444)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
Sorry I did it on one way (Input changing tinymce lang) but not in the other way (Tinymce changing the whole page lang), it's fine now

**Review by PierreRambaud — DISMISSED** [review]

File: admin-dev/themes/new-theme/js/components/tinymce-editor.js [hunk 1]
```diff
@@ -149,17 +157,29 @@ class TinyMCEEditor {
 
       if (translatedField.length && tabContainer.length) {
         const textareaLocale = translatedField.data('locale');
-        const textareaLinkSelector = '.nav-item a[data-locale="'+textareaLocale+'"]';
+        const textareaLinkSelector = '.nav-item a[data-locale="' + textareaLocale + '"]';
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/components/tinymce-editor.js]:
```suggestion
        const textareaLinkSelector = `.nav-item a[data-locale="${textareaLocale}"]`;
```

File: admin-dev/themes/new-theme/js/components/tinymce-editor.js [hunk 2]
```diff
@@ -149,17 +157,29 @@ class TinyMCEEditor {
...
+          EventEmitter.emit('languageSelected', {
+            selectedLocale: textareaLocale,
+            form
+          });
         });
       }
     });
+
+    EventEmitter.on('languageSelected', data => {
+      const textareaLinkSelector = '.nav-item a[data-locale="' + data.selectedLocale + '"]';
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/components/tinymce-editor.js]:
```suggestion
      const textareaLinkSelector = `.nav-item a[data-locale="${data.selectedLocale}"]`;
```

**Review by Progi1984 — COMMENTED** [review]
Some feedback @NeOMakinG

**Review by Progi1984 — COMMENTED** [review]
Don't forget to build assets

**Review by Progi1984 — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by Progi1984** [comment]
@NeOMakinG Could you rebase & rebuild your assets, please ?

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @NeOMakinG

**Pull request merged**